sdk/digitaltwins/digital-twins-core/api-extractor.json

```diff
@@ -11,7 +11,7 @@
   "dtsRollup": {
     "enabled": true,
     "untrimmedFilePath": "",
-    "publicTrimmedFilePath": "./types/digitalTwinsClient.d.ts"
+    "publicTrimmedFilePath": "./types/digital-twins-core.d.ts"
```

Comment:
Calling out this change - wasn't sure the impact so I'd like to have someone 👍 or 👎 this